### ISSUE TITLE
inception update failure - disk busy

### ISSUE THREAD

**Issue by poblin-orange** [orange-cloudfoundry/paas-templates#2099]
### Expected behavior

* As a paas-templates operator
* In order to 
  * ensure smooth upgrade 
* I need 
  * to not have zombie ssh sessions established locking bosh persistent disk (/var/vcap/store) 

### Observed behavior

```
Task 15223851 | 13:45:03 | Preparing package compilation: Finding packages to compile (00:00:00)
Task 15223851 | 13:45:04 | Updating instance inception: inception/47f8df82-3488-4850-929b-9c8b31c0178f (0) (00:10:12)
                     L Error: Action Failed get_task: Task d2d940f2-7ce2-4750-5732-aaf178d8bbd0 result: Unmounting persistent disk: Running command: 'umount /dev/sdc1', stdout: '', stderr: 'umount: /var/vcap/store: target is busy.
': exit status 32

{"time":1696340707,"stage":"Updating instance","tags":["inception"],"total":1,"task":"inception/47f8df82-3488-4850-929b-9c8b31c0178f (0)","index":1,"state":"in_progress","progress":40,"data":{"status":"executing post-stop"}}
{"time":1696341316,"stage":"Updating instance","tags":["inception"],"total":1,"task":"inception/47f8df82-3488-4850-929b-9c8b31c0178f (0)","index":1,"state":"failed","progress":100,"data":{"error":"Action Failed get_task: Task d2d940f2-7ce2-4750-5732-aaf178d8bbd0 result: Unmounting persistent disk: Running command: ''umount /dev/sdc1'', stdout: '''', stderr: ''umount: /var/vcap/store: target is busy.\n'': exit status 32"}}
{"time":1696341316,"error":{"code":450001,"message":"Action Failed get_task: Task d2d940f2-7ce2-4750-5732-aaf178d8bbd0 result: Unmounting persistent disk: Running command: ''umount /dev/sdc1'', stdout: '''', stderr: ''umount: /var/vcap/store: target is busy.\n'': exit status 32"}}
```

during the upgrade, if ssh session are established on the inception vm, the bosh disk unmount fails. Might be most frequent with tmux usage ?

```
incepti+ 31744 31743  0 Sep13 ?        00:00:00 (sd-pam)
inception/47f8df82-3488-4850-929b-9c8b31c0178f:~# lsof |grep /var/vcap/store/
bosh       2131                  inception  cwd       DIR               8,33     4096    1179649 /var/vcap/store/inception
bosh       2131                  inception  txt       REG               8,33 36318973    1179658 /var/vcap/store/inception/bin/bosh
bosh       2131                  inception    8u      REG               8,33     1675    1189937 /var/vcap/store/inception/.bosh/tmp/ssh-priv-key4036381546
bosh       2131                  inception    9u      REG               8,33      589    1189955 /var/vcap/store/inception/.bosh/tmp/ssh-known-hosts3658320992
bosh       2131  2132            inception  cwd       DIR               8,33     4096    1179649 /var/vcap/store/inception
bosh       2131  2132            inception  txt       REG               8,33 36318973    1179658 /var/vcap/store/inception/bin/bosh
bosh       2131  2132            inception    8u      REG               8,33     1675    1189937 /var/vcap/store/inception/.bosh/tmp/ssh-priv-key4036381546
bosh       2131  2132            inception    9u      REG               8,33      589    1189955 /var/vcap/store/inception/.bosh/tmp/ssh-known-hosts3658320992
bosh       2131  2133            inception  cwd       DIR               8,33     4096    1179649 /var/vcap/store/inception
bosh       2131  2133            inception  txt       REG               8,33 36318973    1179658 /var/vcap/store/inception/bin/bosh
bosh       2131  2133            inception    8u      REG               8,33     1675    1189937 /var/vcap/store/inception/.bosh/tmp/ssh-priv-key4036381546
bosh       2131  2133            inception    9u      REG               8,33      589    1189955 /var/vcap/store/inception/.bosh/tmp/ssh-known-hosts3658320992
bosh       2131  2134            inception  cwd       DIR               8,33     4096    1179649 /var/vcap/store/inception

```
 
```
systemd─┬─VGAuthService
        ├─agetty
        ├─runsvdir─┬─runsv─┬─bosh-agent─┬─post-deploy───bash───bash───bash───bash───bash─┬─bash───make───sh───ruby
        │          │       │            │                                                ├─bash───awk
        │          │       │            │                                                └─tee
        │          │       │            └─11*[{bosh-agent}]

```

NB: tmux session from inception also induced a disk busy on 00-core-connectivity  due to a bosh ssh targeting /var/vcap/store

```
inception/47f8df82-3488-4850-929b-9c8b31c0178f:~# ps -ef |grep bosh
...
incepti+  2131 28576  0 Sep28 pts/8    00:00:23 bosh ssh server/0
incepti+ 24814 17094  0 Sep26 pts/5    00:00:34 bosh ssh mysql/0
```

### Affected releases

* 56

### Workaround

reboot vm and relaunch upgrade

**Comment by JCL38-ORANGE** [comment]
Issue upstream to be done

**Comment by ogrand** [comment]
Fix with 56.0.1
